identity-apps-core/apps/recovery-portal/src/main/webapp/self-registration-process.jsp

```diff
@@ -226,8 +227,22 @@
                 }
             }
             request.getRequestDispatcher("register.do").forward(request, response);
+        } else if (!skipSignUpEnableCheck 
+            && SelfRegistrationStatusCodes.ERROR_CODE_USER_ALREADY_EXISTS.equalsIgnoreCase(errorCode)) {
+            if (isAccountVerificationEnabled && !isShowUsernameUnavailabilityEnabled) {
+                request.setAttribute("callback", callback);
+                if (StringUtils.isNotBlank(srtenantDomain)) {
+                    request.setAttribute("srtenantDomain", srtenantDomain);
```

Comment:
what does `sr` prefix mean in the param name? shall we add an inline comment to clarify that maybe?